src/ILCompiler.Compiler/src/CppCodeGen/ILToCppImporter.cs

```diff
@@ -1652,6 +1776,7 @@ private void ImportStoreIndirect(TypeDesc type)
 
             // TODO: Write barrier as necessary!!!
 
+            AppendLine();
```

Comment:
There extra lines will throw off the lines generated by `#pragma line` directives - so the stepping in C# code under debugger will not work well anymore. I think we may need different modes: one that produces nice readable C++, and other that has the `#pragma line` directives in it, but it is less readable.